src/coprocessor/codec/data_type/vector.rs

```diff
@@ -91,6 +92,16 @@ impl VectorValue {
         }
     }
 
+    pub fn swap(&mut self, from_row: usize, to_row: usize) {
```

Comment:
Please don't implement `swap`, since it breaks the character that data are supposed to be stored continuously. `VectorValue` and `LazyBatchColumn` should ensure the possibility of sequential modification (e.g. push only, or pop only, or truncate). Please think `VectorValue` and `LazyBatchColumn` as something like a `Vec<u8>` or `Chunk` where all elements are stored one by one compactly (because it is what it will become finally). Random position modifications should be simply forbidden since it's cost is too high in such scenario.